tests/dataset_tests/parsers_tests/test_sdf_file_parser.py

```diff
@@ -96,15 +115,34 @@ def test_sdf_file_parser_target_index(sdf_file, mols):
     assert smiles[1] == 'CC1=CC2CC(CC1)O2'
 
 
+def test_sdf_file_parser_return_is_successful(sdf_file_long, mols):
+    """test `labels` option and retain_smiles=True."""
+    preprocessor = NFPPreprocessor(max_atoms=10)
```

Comment:
specifying max_atoms=10 to see if `is_successful` work correctly or not.
I know that this depends on the `preprocessor` behavior, which is not good way of unittesting...